src/System.Drawing.Common/src/mono/MonoTODOAttribute.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
There are quite a few usages in the mono code. Since we'd like to share the source files from here, I left the attributes in so that mono could still compile with them. Instead of #if'ing them out from the .NET Core version, I just defined a dummy attribute.